src/content/subgames.cpp

```diff
@@ -134,8 +134,67 @@ SubgameSpec findSubgame(const std::string &id)
 		}
 	}
 
-	if (game_path.empty())
-		return SubgameSpec();
+	std::string idv = id;
+
+	// Failed to find the game, try to find aliased game
+	if (game_path.empty()) {
+		std::vector<GameFindPath> gamespaths;
+		gamespaths.emplace_back(share + DIR_DELIM + "games", false);
+		gamespaths.emplace_back(user + DIR_DELIM + "games", true);
+
+		Strfnd search_paths(getSubgamePathEnv());
+
+		while (!search_paths.at_end())
+			gamespaths.emplace_back(search_paths.next(PATH_DELIM), false);
+
+		for (const GameFindPath &gamespath : gamespaths) {
+			const std::string &path = gamespath.path;
+			std::vector<fs::DirListNode> dirlist = fs::GetDirListing(path);
+			for (const fs::DirListNode &dln : dirlist) {
+				if (!dln.dir)
+					continue;
+
+				// If configuration file is not found or broken, ignore game
+				Settings conf;
+				std::string conf_path = path + DIR_DELIM + dln.name +
+							DIR_DELIM + "game.conf";
+				if (!conf.readConfigFile(conf_path.c_str()))
+					continue;
+
+				if (conf.exists("gameid_alias")) {
+					std::vector<std::string> aliases = str_split(conf.get("gameid_alias"), ',');
+					for (const std::string &alias_raw : aliases) {
+						std::string_view alias = trim(alias_raw);
+						auto found_alias = [&](){
+							idv = dln.name;
+							game_path = path + DIR_DELIM + dln.name;
+							user_game = gamespath.user_specific;
+						};
+						// Try direct matching with the alias
+						if (alias == id) {
+							found_alias();
+							break;
+						// Make sure a "_game" suffix is ignored
```

Comment:
Same here: Aren't you missing the case where both end in `_game`? Again I think a "normalize" function / lambda which trims off the `_game` and returns a string (view) would make sense. Then you could just compare the appropriate "normalized" string views.